src/Evolution/Systems/CurvedScalarWave/Initialize.hpp

```diff
@@ -0,0 +1,133 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <tuple>
+#include <utility>
+
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/EagerMath/Norms.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/Tags.hpp"
+#include "Evolution/Initialization/InitialData.hpp"
+#include "Evolution/Systems/CurvedScalarWave/System.hpp"
+#include "NumericalAlgorithms/Spectral/Mesh.hpp"
+#include "Parallel/ArrayIndex.hpp"
+#include "Parallel/GlobalCache.hpp"
+#include "Parallel/ParallelComponentHelpers.hpp"
+#include "Parallel/PhaseDependentActionList.hpp"
+#include "ParallelAlgorithms/Initialization/MutateAssign.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Christoffel.hpp"
+#include "PointwiseFunctions/GeneralRelativity/GeneralizedHarmonic/ExtrinsicCurvature.hpp"
+#include "Time/Time.hpp"
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+namespace CurvedScalarWave::Actions {
+/// \ingroup InitializationGroup
+/// \brief Initialize items adding constraint damping parameters of the
+/// CurvedScalarWave system
+///
+/// DataBox changes:
+/// - Adds:
+///   * `CurvedScalarWave::Tags::ConstraintGamma1`
+///   * `CurvedScalarWave::Tags::ConstraintGamma2`
+/// - Removes: nothing
+/// - Modifies: nothing
+///
+/// \note This action relies on the `SetupDataBox` aggregated initialization
+/// mechanism, so `Actions::SetupDataBox` must be present in the
+/// `Initialization` phase action list prior to this action.
+
+template <size_t Dim>
+struct InitializeConstraintDampingGammas {
```

Comment:
It would be nice to have tests for these actions. You can make your life a lot simpler, though! :D Nobody likes writing action tests (let's be real here, right???!!) and so you can actually turn these two actions into two mutators. That means you can have tests without the action testing framework (yay!) because you only need to deal with a `DataBox`. Here's an example mutator https://github.com/sxs-collaboration/spectre/blob/develop/src/Evolution/Systems/GrMhd/ValenciaDivClean/SetVariablesNeededFixingToFalse.hpp and here's a more complicated one https://github.com/sxs-collaboration/spectre/blob/develop/src/Evolution/Systems/GrMhd/ValenciaDivClean/Subcell/GrTagsForHydro.hpp You can use these with the https://github.com/sxs-collaboration/spectre/blob/develop/src/ParallelAlgorithms/Initialization/Actions/AddSimpleTags.hpp and https://github.com/sxs-collaboration/spectre/blob/develop/src/ParallelAlgorithms/Initialization/Actions/AddComputeTags.hpp actions in the `EvolveCurvedScalarWave.hpp` file. Testing the mutators will be fairly painless, hopefully :)